SwissTransferFeatures/TransferList/TransferList.swift

```diff
@@ -23,17 +23,20 @@ import SwiftUI
 import SwissTransferCore
 import SwissTransferCoreUI
 
-public struct TransferList: View {
+public struct TransferList<EmptyView: View>: View {
     @Environment(\.isCompactWindow) private var isCompactWindow
     @EnvironmentObject private var mainViewState: MainViewState
 
     @StateObject private var viewModel: TransferListViewModel
 
     private let origin: TransferOrigin
 
-    public init(transfers: [TransferUi], origin: TransferOrigin) {
-        _viewModel = StateObject(wrappedValue: TransferListViewModel(transfers: transfers))
+    public let emptyView: EmptyView?
+
+    public init(transferManager: TransferManager, origin: TransferOrigin, @ViewBuilder emptyView: @escaping () -> EmptyView) {
```

Comment:
As we call the `emptyView` method directly in the init, is it really `@escaping` ?